hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/CassDefinitionsServiceImpl.java

```diff
@@ -95,6 +95,9 @@
     private final MsgLogger msgLog = MsgLogger.LOGGER;
     private final Logger log = Logger.getLogger(CassDefinitionsServiceImpl.class);
 
+    private Set<DefinitionsEvent> deferredNotifications = new HashSet<>();
```

Comment:
Well, I think it could create confusion as this state is used internally in importDefinitions() method who performs some calls, but the shared variable is managed at EJB stateless instance.
So yes, if you can add some comments I guess it will help in the future just if we wonder why we put it there.
Thanks.